uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/ActivityBeansCache.java

```diff
@@ -296,6 +323,9 @@ private boolean activitySupportsPath(ActivityAndMetaInfo activity, Path path) {
         return new ArrayList<String>(activitiesById.keySet());
     }
 
+    public void noOp() {
```

Comment:
A small comments inside of this method explaining it could help with readiability

Reply:
// Does nothing, simulates...